test/fixtures/mson-mixin-nonexistent.apib

```diff
@@ -9,4 +9,4 @@
 
 ## User (object)
 - id
-- Include Address
+- Include XXX
```

Comment:
> with older version of snowcrash it was working

Yes, but I added this check during the bug-fixes.

> In future it should work again

Yes, but that's in the future.